jproject-fr/next-drupal-starterkit
# Added the JSON-API-EXTRAS module to drupal

# DONE

- Added this module to drupal
https://www.drupal.org/project/jsonapi_extras

- It allows us to only return the desired field in the api in order to have a cleaner one

# TO TEST

- Open the Drupal in the admin part and in the module list you should see the JSON-API-EXTRAS modules
